installer/requirements.in

```diff
@@ -19,6 +19,7 @@ torch-fidelity
 torchvision==0.13.1 ; platform_system == 'Darwin'
 torchvision==0.13.1+cu116 ; platform_system == 'Linux' or platform_system == 'Windows'
 transformers
+picklescan==0.0.5
```

Comment:
We shouldn't pin this in here (let pip-compile figure it out), UNLESS there's some hard requirement to do so